internal/apps/dop/providers/issue/core/service.go

```diff
@@ -616,25 +612,30 @@ func (i *IssueService) BatchDeleteIssues(ctx context.Context, req *pb.BatchDelet
 		// 删除测试计划用例关联
 		if issue.Type == pb.IssueTypeEnum_BUG {
 			if err := i.db.DeleteIssueTestCaseRelationsByIssueIDs([]uint64{numSlice[k]}); err != nil {
```

Comment:
I think tx not works here.

Reply:
you should pass tx as args.

Reply:
ok，i reinitialized the parameters using tx